File: 1036.py
# URI 1036 em Python 3.9
# Programador: Matheus Felipe Alves Durães
# Não copie códigos, apenas leia-os e
# tente entender O QUE e POR QUE fazem o que fazem. 

# DA biblioteca math IMPORTAR sqrt (raiz quadrada)
from math import sqrt 

def main():
    # Recebe os valores A B C para possivelmente formar um triangulo
    a, b, c = map(float, input().split())

    # Aqui calculamos a primeira parte do delta sem a raiz quadrada
    delta = b**2 - 4*a*c

    # Fazemos uma verificação simples se o delta é menor ou igual a 0
    # OU se A é igual a zero, pois se for também se torna impossivel calcular
    if delta <= 0 or a <= 0:
        print("Impossivel calcular")
    
    else:
        # Caso não seja impossivel de calcular então agora fazemos
        # a raiz quardade de delta
        delta = sqrt(delta)

        # Calculamos o R1 e o R2
        r1 = (-b + delta)/(2*a)
        r2 = (-b - delta)/(2*a)

        # Printando os dois resultados
        print("R1 = %.5f" %(r1) )
        print("R2 = %.5f" %(r2) )

main()

File: C/1000-1999/1036.c
#include <stdio.h>
#include <math.h>

int main (void){
  double a, b, c, r1, r2, sq;

  scanf("%lf %lf %lf", &a, &b, &c);

  sq = b*b - 4*a*c;

  if (sq <= 0 || a == 0){
    printf("Impossivel calcular\n");
  } else {
    r1 = (b*-1 + sqrt(sq))/(2*a);
    r2 = (b*-1 - sqrt(sq))/(2*a);

    printf("R1 = %.5lf\n", r1);
    printf("R2 = %.5lf\n", r2);
  }

  return 0;
}
